### PR TITLE
chore(AutoSuggest): Remove unused Autosuggest impl

### PR DESCRIPTION
**Pull request by lordkiz**
### Description
- Removes the unused Autosuggest implementation for trailing inline Input suggestions

This PR resolves []

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Remove unused Autosuggest - kizito

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/shared/utils/normalizeText.ts [hunk 1]
```diff
@@ -241,10 +241,10 @@ for (const { letters, base } of defaultDiacriticsRemovalMap) {
 
 // "what?" version ... http://jsperf.com/diacritics/12
 export function normalizeText(text: string | null, trim: boolean = true) {
-  text = text || ""
-  text = trim ? text.trim() : text
+  let textString = text || ""
```
Comment (dblandin) [inline src/shared/utils/normalizeText.ts:244]: Just to confirm - this utility is used and suffers from the same problem? Have we encountered a bug referencing an unknown keyword of `text`?

**Review by dblandin — APPROVED** [review]
LGTM. Left one clarifying question.

File: src/shared/utils/normalizeText.ts (hunk 1 above)
Comment (lordkiz) [reply src/shared/utils/normalizeText.ts:244]: To the best of my knowledge, we have not. 🤔

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
Thanks for the fix Kizito. Let's cherry pick this into the upcoming release

**Pull request merged**